src/app/components/common/topbar/topbar.component.ts

```diff
@@ -148,14 +145,25 @@ export class TopbarComponent extends ViewControllerComponent implements OnInit,
         this.checkNetworkChangesPending();
       }
     });
-    this.continuosStreaming = observableInterval(10000).subscribe(x => {
-      this.showReplicationStatus();
-      if (this.is_ha) {
-        this.getHAStatus();
-      }
-      this.checkNetworkCheckinWaiting();
-      this.checkNetworkChangesPending();
-    });
+    this.replicationStatusSub = this.ws
```

Comment:
this should not replace the continous streaming, but should be outside of it, the getHAStatus needs to still be in there until we convert it to use events instead of check every 10 seconds